chatterbot/adapters/logic/time_adapter.py

```diff
@@ -0,0 +1,32 @@
+from .logic import LogicAdapter
+from chatterbot.conversation import Statement
+from textblob.classifiers import NaiveBayesClassifier
+from datetime import datetime
+
+
+class TimeLogicAdapter(LogicAdapter):
+
+    def __init__(self, **kwargs):
+        super(TimeLogicAdapter, self).__init__(**kwargs)
+
+        training_data = [
+            ("what time is it", 1),
+            ("do you know the time", 1),
+            ("do you know what time it is", 1),
+            ("what is the time", 1),
+            ("do you know the time", 0),
+            ("it is time to go to sleep", 0),
+            ("what is your favorite color", 0),
+            ("i had a great time", 0)
+        ]
+
+        self.classifier = NaiveBayesClassifier(training_data)
```

Comment:
Sorry for the confusing message. I am referring to the NaiveBayesClassifier. I would create different classifiers, such as a NaiveBayesClassifier, a neural network classifier, etc, and then call those from within the logic adapters. This allows for flexibility & ease of updating if, for example, you have a lot of logic adapters that use the same code. If that code needs an update, it would be simpler and cleaner to just have to update the code in a single location rather than many locations.